server/database/seeds/06_quizzes.js

```diff
@@ -1,13 +1,9 @@
-
 exports.seed = function(knex, Promise) {
   // Deletes ALL existing entries
-  return knex('table_name').del()
-    .then(function () {
+  return knex('quizzes')
+    .del()
+    .then(function() {
       // Inserts seed entries
-      return knex('table_name').insert([
-        {id: 1, colName: 'rowValue1'},
-        {id: 2, colName: 'rowValue2'},
-        {id: 3, colName: 'rowValue3'}
-      ]);
+      return knex('quizzes').insert([{ quiz_type: 'test' }, { quiz_type: 'learn' }, { quiz_type: 'test' }]);
```

Comment:
dupe seed for test

Reply:
i think its ok in this case, because there are going to be more than two quizzes in the database. In this case i made 2 dummy test quizzes to add questions to and one learn quiz to add to as well.